Ska/ParseCM.py

```diff
@@ -172,4 +173,28 @@ def read_mm(filename):
     return manvr_list
 
 
+def read_si_align(char_file):
+    """
+    Read ODB_SI_ALIGN matrices from supplied characteristics file
 
+    :param char_file: OFLS ODB characteristics file
+    :returns: dictionary of matrices, one per detector in original Fortran order
+    """
+    char_lines = open(char_file).readlines()
+    # Find the beginning of the assignment using the regex from aimpoint_mon
+    RE_ODB_SI_ALIGN = re.compile(r'\s* ODB_SI_ALIGN \s* =', re.VERBOSE)
+    matches = [i for i, line in enumerate(char_lines) if RE_ODB_SI_ALIGN.match(line)]
+    if len(matches) != 1:
+        raise ValueError('parsing error, matched {} instances of ODB_SI_ALIGN instead of one'
+                         .format(len(matches)))
+    start = matches[0]
+    # Grab the 12 lines of the alignment.  This assumes no extra blank lines
+    odb_si_lines = char_lines[start:start+12]
+    # Matching the values using a simple comma/space pattern with very little checking
+    odb_strings = [list(re.search("(\S+),\s(\S+),\s(\S+),", line).groups()) for line in odb_si_lines]
```

Comment:
It's easy enough to use the float regex from https://docs.python.org/2/library/re.html#simulating-scanf, but I see your point in not overcomplicating things.  But since this is a routine that needs to survive 15 years, it can be a little bit nicer and harder against slight future format changes.  Use `re.VERBOSE` and `\s* , \s*` for separators.  (Not sure if the ODB allows trailing space before the comma, but it can't hurt).